tests/main/upgrade-from-2.15/task.yaml

```diff
@@ -3,10 +3,18 @@ summary: Ensure upgrades from snapd 2.15 work
 systems: [ubuntu-16.04-64]
 
 prepare: |
-    dpkg --purge snapd
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB/pkgdb.sh"
+    distro_purge_package snapd
 
 restore: |
-    dpkg --purge ubuntu-core-launcher snap-confine
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB/pkgdb.sh"
+    if [ -e old_install_done ]; then
+        echo "Ensure core is gone and we have ubuntu-core instead"
+        distro_purge_package snapd
+    fi
+    distro_install_build_snapd
```

Comment:
Just a nitpick, this function name should probably say `distro_install_built_snapd`.

Reply:
@zyga agree, I'll make that change in a following PR due it is being used in several tests. Thanks for reviewing!!